procedures/katello/branch.md.erb

```diff
@@ -18,10 +18,9 @@
   - [ ] [Install grunt](https://github.com/Katello/katello/blob/master/engines/bastion_katello/README.md)
   - [ ] `grunt i18n:extract` in the katello/engines/bastion_katello directory
   - [ ] `tx pull --minimum-perc 50 --all` in the katello directory
-  - [ ] `make -C locale tx-pull` in the katello directory
+  - [ ] `make -C locale tx-update` in the katello directory
```

Comment:
Technically the only difference between `tx-pull` and `tx-update` is that `tx-update` also warns you to do a follow up step in the foreman directory. That's listed as a step 2 lines below so I think `tx-pull` is the better choice
```suggestion
  - [ ] `make -C locale tx-pull` in the katello directory
```

Reply:
Sounds good, I put it back to tx-pull